src/wallet.js

```diff
@@ -162,7 +164,7 @@ const wallet = {
     let promise = null;
     if (loadHistory) {
       // Load history from address
-      WebSocketHandler.setup();
+      this._connection.setup();
```

Comment:
Besides the other comment about being aware of the Connection class, we're changing it to have just 1 ws connection. In that case, using the singleton seems even more appropriate.

Reply:
Discussed this on a call. `WebSocketHandler` is the default if no `Connection` is set.